lib/service/database.js

```diff
@@ -339,6 +374,10 @@ DatabaseService.prototype.queueAnalysisOperations = function(analysis, callback)
             }
         });
 
+        asyncQueries.push({
+            query: invalidateCache(Object.keys(affectedTables))
```

Comment:
That was my point, to avoid invalidating for every node but it seems like is not a good point at all. So moving do execute the invalidation for every node